src/commands/gaming/currency/balance.ts

```diff
@@ -5,17 +5,18 @@ import { createCommand } from "../../../utils/helpers.ts";
 createCommand({
   name: "balance",
   aliases: ["bal", "coins"],
+  arguments: [{ type: "snowflake", name: "user", required: false }],
```

Comment:
I would make this a `member` and then just use member.id below. This will validate the snowflake string and also not allow someone to abuse and get access to member details who they dont share a server with

Reply:
Yes you're right